test/default-messages.js

```diff
@@ -1,37 +1,32 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('default messages', function (t) {
     t.plan(1);
 
-    var tc = tap.createConsumer();
+    var ps = spawn(process.execPath, [ __dirname + '/messages/defaults.js' ]);
```

Comment:
The Windows OS treats / as \ in cases like this, so Node chooses to normalize since doing so has no effect besides making the string consistent. (POSIX does not act symmetrically.)